cmd/configurator/BUILD.bazel

```diff
@@ -9,7 +9,12 @@ go_binary(
     pure = "on",
 )
 
-make_image(name = "image")
+make_image(
+    name = "image",
+    directory = "/",
+    files = [":configurator"],
```

Comment:
Why? This isn't what we should be doing and this isn't how make_image works (note that it embeds :go_default_library and builds it differently.

This breaks macos builds.